pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -1772,7 +1765,7 @@ status:
 			}, {
 				Operation: "add",
 				Path:      "/spec/statusMessage",
-				Value:     string(v1alpha1.RunCancelledByPipelineMsg),
```

Comment:
can this constant be removed?

Reply:
I feel It's better if we keep it same as other test cases. Oh actually, we may want to update the previous one as `v1alpha1.RunReasonCancelled`, nice catch!

Reply:
sorry, I meant can `RunCancelledByPipelineMsg` be removed from the v1alpha1 package?

Reply:
Oh yes, you are totally right, sorry I should've asked. We'll need to move all related usage of `v1alpha1.Run` to `v1beta1` later on as part of https://github.com/tektoncd/pipeline/issues/5153, added https://github.com/tektoncd/pipeline/issues/5153#issuecomment-1290866807 to make sure this will happen.